### PR TITLE
Add wrapper for Nathan Boleu's IDR package.

### PR DESCRIPTION
**Pull request by davebx**

### CODE REVIEW

File: tools/idr/idr.xml [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        $allow_negative_scores --random-seed $random_seed --max-iter $max_iter
+        --convergence-eps $convergence_eps $only_merge_peaks
+        ]]>
+    </command>
+    <inputs>
+        <param name="primus" argument="--samples" type="data" format="bed,gff" label="First set of peaks in GFF or BED format" />
+        <param name="secundus" argument="--samples" type="data" format="bed,gff" label="Second set of peaks in GFF or BED format" />
+        <param name="peak_list" argument="--peak-list" type="data" format="bed,gff" optional="True" label="List of peaks to filter by" />
+        <param name="input_file_type" argument="--input-file-type" type="select" label="Input BED format" help="This option is ignored if the input file is GFF">
+            <option value="" selected="true">Vanilla BED</option>
```
Comment (bgruening) [inline tools/idr/idr.xml]: Two are selected=True ... this is probably a mistake?

**Comment by bgruening** [comment]
Thanks Dave!

File: tools/idr/idr.xml (hunk 1 above)
Comment (davebx) [inline tools/idr/idr.xml]: Definitely a mistake. Looking into why there are test failures right now, everything is passing on my local system.

Comment (nsoranzo) [inline tools/idr/idr.xml]: @davebx The PR was merged, I don't think you need to fix anything.

**Pull request merged**